charts/vector/values.yaml

```diff
@@ -402,6 +435,9 @@ podMonitor:
   # podMonitor.honorTimestamps -- If true, honor_timestamps is set to true in the [scrape config](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config).
   honorTimestamps: true
 
+# Log level for Vector. Default is "info".
```

Comment:
```suggestion
# Log level for Vector.
```

The readme generated with `helm-docs` should show the default value.

Reply:
thx for the review, fixed!